src/debugger/debugAdapterFactory.ts

```diff
@@ -77,9 +77,14 @@ export class LLDBDebugConfigurationProvider implements vscode.DebugConfiguration
     ): Promise<vscode.DebugConfiguration> {
         launchConfig.env = this.convertEnvironmentVariables(launchConfig.env);
         // Fix the program path on Windows to include the ".exe" extension
-        if (this.platform === "win32" && path.extname(launchConfig.program) !== ".exe") {
+        if (
+            this.platform === "win32" &&
+            launchConfig.testType === undefined &&
```

Comment:
This works for now, but it would be nice to have a more "universal" way to determine if the `.exe` suffix needs to be added. I didn't anticipate this use case when I added this code. It could definitely use some work.

Perhaps we can have the auto-generated launch configurations use a `target` parameter instead of `program`, for example. That way the launch config can be used across all platforms while retaining the ability to target a specific executable for the tests.

I'm going to open up a separate issue from this since this needs to go in to unblock test debugging on Windows.